### DISCUSSION TITLE
watch wont connect to server

### DISCUSSION THREAD

**Discussion by PeppermintPubby** [200Tigersbloxed/FitbitHRtoWS#15] [Q&A]
I know the servers no longer worked on
but i have a fitbit versa 2, if anyone see's this and could help id be very thankful <3

**Comment by 200Tigersbloxed** [comment]
The global server was removed a while ago. You can use the HRProxy app instead if you don't want to self-host, or use a different provider.

[HRProxy SDK4](https://gallery.fitbit.com/details/2ee1ee3a-7d8d-4c87-988c-77f542370610)
[HRProxy SDK6](https://gallery.fitbit.com/details/d05e9ed4-a29b-45b3-888c-5ddafd9b3c99)

**Reply by PeppermintPubby** [reply]
Hi! Sorry to bother again 
But im still having issues connecting the watch to hrtovrc app, i installed the above sdk and got the app to connect to server. Any help is appreciated!
